Header header

uint32 time
uint32 time_ns
uint8 sync_flag
